package.json

```diff
@@ -113,7 +113,7 @@
   "bundlesize": [
     {
       "path": "./dist/reflex.min.js",
-      "maxSize": "6.0 Kb"
```

Comment:
I was going to, but readme looks like its already set to 7.0kb which is why I didn't change it. Also why I went with 7k for the maxSize